tests/php/Controllers/CMSMainTest.php

```diff
@@ -125,7 +125,7 @@ public function testPublish()
     {
         $page1 = $this->objFromFixture(Page::class, "page1");
         $page2 = $this->objFromFixture(Page::class, "page2");
-        $this->session()->set('loggedInAs', $this->idFromFixture(Member::class, 'admin'));
+        $this->logInAs($this->idFromFixture(Member::class, 'admin'));
```

Comment:
```suggestion
        $this->logInAs('admin');
```
There's a call to `$this->session()->clear('loggedInAs');` on line 151 which presumably should be a logout ... or maybe not exists at all.

Reply:
I've removed the `$this->session()->clear('loggedInAs');`

`$this->logInAs($this->idFromFixture(Member::class, 'admin'));` should be kept as is because there's a specific fixture that's being used.  Elsewhere in the file `$this->logInWithPermission('ADMIN');` is used which doesn't use that specific fixture

Reply:
What about this one?

Reply:
Done